@xen-orchestra/web-core/lib/components/tab/TabList.vue

```diff
@@ -6,17 +6,13 @@
 </template>
 
 <script lang="ts" setup>
-import { useContext } from '@core/composables/context.composable'
-import { DisabledContext } from '@core/context'
+import { useDisabled } from '@core/composables/disabled.composable'
 
-const props = withDefaults(
-  defineProps<{
-    disabled?: boolean
-  }>(),
-  { disabled: undefined }
-)
+const props = defineProps<{
+  disabled?: boolean
+}>()
```

Comment:
Why use prop destruction for some components and not others?

```suggestion
const {  disabled } = defineProps<{
  disabled?: boolean
}>()
```

Reply:
> Why use prop destruction for some components and not others?

This PR reflects code written in November, before we decided to use props destructuring.

The new files that appeared during my last rebase on master were handled with destructuring since that's our new standard.

But converting all existing files to use destructuring will be handled in a separate PR.